components/datatypes/media.schema.json

```diff
@@ -236,7 +236,7 @@
                 "xdm:videoSegment": {
                   "title": "Video segment",
                   "type": "string",
-                  "pattern": "^[[0-9]+-[0-9]+]$",
+                  "pattern": "^[0-9]+-[0-9]+$",
```

Comment:
From our previous experience it won't work regardless of the pattern until the pattern is not removed. At some point as a test I created a field with a very simple pattern(to contain a digit) at that one didn't work either. So I presumed that it is a general problem regarding the pattern property of a field and it is not working as expected.